doors_dashboards/components/scattermap.py

```diff
@@ -133,4 +145,35 @@ def set_feature_handler(self, feature_handler: FeatureHandler):
         self.feature_handler = feature_handler
 
     def register_callbacks(self, app: Dash, component_ids: Dict[str, str]):
-        pass
+        @app.callback(
+            Output("general", "data"),
+            Input("scattermap", 'clickData'),
```

Comment:
I know that I introduced the strings "general" and "scattermap", but we should strive to making them constants somewhere. "general" can be a constant in constant.py, "scattermap" should be one in the scattermap module